.github/workflows/test-pr.yml

```diff
@@ -0,0 +1,47 @@
+name: Test PR
+
+on: pull_request
+
+jobs:
+  build:
+    name: Build
+    uses: alphagov/di-onboarding-product-page/.github/workflows/build.yml@f750c28d35c2dce51c78ad36b6e112e8cf0ec62b
```

Comment:
Out of curiosity, how do you know the `@tag...` is it a commit hash?

Reply:
[GHA docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iduses)

> `{ref}` can be a SHA, a release tag, or a branch name. Using the commit SHA is the safest for stability and security.

At the moment this is the SHA of the first commit in this PR (that's why I need two commits).
Once this is merged I will change this to refer to the SHA from main or maybe just `@main` so we use the latest merged version.

Reply:
Nice. Thanks.